src/request/util.ts

```diff
@@ -7,14 +7,13 @@ import UrlSearchParams from '../UrlSearchParams';
  * @param url The base URL.
  * @param options The options hash that is used to generate the query string.
  */
-export function generateRequestUrl(url: string, options: RequestOptions): string {
+export function generateRequestUrl(url: string,
+		{ cacheBust }: RequestOptions = {}): string {
```

Comment:
`query` needs to be de-structured as well to avoid the issue in line 12 which the compiler is complaining about, because now there is no `options`